opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -1743,7 +1751,7 @@ namespace Opm {
 
 
     template<typename TypeTag>
-    std::pair<bool, bool>
+    std::tuple<bool, bool, typename BlackoilWellModel<TypeTag>::Scalar>
     BlackoilWellModel<TypeTag>::
```

Comment:
Cant you just use Scalar here?

Reply:
I believe I tried that first, and it didn't compile at my end (GCC13)

Reply:
Scalar has no meaning in the context as the return value is defined "outside" the class context. typename is required.

Reply:
Thanks. I see that now. @vkip can you undraft this if you think it is ready for merging.